package.json

```
@@ -70,6 +71,26 @@
         "command": "runme.resetEnv",
         "category": "Runme",
         "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.resetEnv",
+        "category": "Runme",
+        "title": "Reset Environment Variables for current VS Code session"
+      },
+      {
+        "command": "runme.openAsRunmeNotebook",
+        "category": "Runme",
+        "title": "Open markdown as runme",
+        "icon": {
+          "light": "assets/runme-logo-light.svg",
+          "dark": "assets/runme-logo-dark.svg"
+        }
+      },
+      {
+        "command": "runme.openSplitViewAsMarkdownText",
+        "category": "Runme",
+        "title": "Open runme notebook as text editor",
```

(non native english speaker here 👋 ) I am not sure but doesn't make this more sense
```suggestion
        "title": "Open Runme notebook in text editor",
```